### PR TITLE
#hacktoberfest feat: change from require to import based syntax #20

### PR DESCRIPTION
**Pull request by Kunalbhandari**
Signed-off-by: kunal bhandari <kunalbhandari15@gmail.com>

### CODE REVIEW

File: run.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import parse from './lib/index.js';
```
Comment (nsinghal12) [inline run.js:1]: This looks like test code and is not required as part of this PR. Please remove this file from PR.

File: test/index.js [hunk 2]
```diff
@@ -57,30 +60,30 @@ describe('require("pom-parser")', function () {
...
       });
     });
   });
 
   describe('when opts is empty', function(){
     it('parser should throw an error', function(){
-      assert.throws(function(){ 
-        pomParser.parse({}, function(err, response){}, Error);
+      assert.throws(function(){
+        parse(null, function(err, response){}, Error);
```
Comment (nsinghal12) [inline test/index.js:71]: Is this change of changing argument from `{}` to `null` required?

**Review by nsinghal12 — CHANGES_REQUESTED** [review]

File: run.js (hunk 1 above)
Comment (Kunalbhandari) [reply run.js]: removed

File: test/index.js (hunk 2 above)
Comment (Kunalbhandari) [reply test/index.js]: i think it was a typo, removed it.

**Comment by nsinghal12** [comment]
Thanks for addressing PR comments. Can you please make sure that you sign off your commits.

**Comment by Kunalbhandari** [comment]
done @nsinghal12 signed commits pushed
also could add hacktober and hacktoberfest-accepted labels.

Thanks

**Review by nsinghal12 — APPROVED** [review]
Looks good to me. Thanks!

**Pull request merged**